src/nodedef.cpp

```diff
@@ -447,7 +448,12 @@ void ContentFeatures::serialize(std::ostream &os, u16 protocol_version) const
 		}
 	}
 	writeU8(os, param_type);
-	writeU8(os, param_type_2);
+	if (protocol_version < 41 && param_type_2 == CPT2_COLORED_FLOWINGLIQUID) {
```

Comment:
Should be mentioned in `networkprotocol.cpp`. Also needs to be bumped to 47 (corresponding to 5.11).